app/components/navBar/navBar.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * Created by gaston on 8/8/16.
+ */
+/**
+ *
+ * @constructor
+ */
+function NavBar() {
+  var self = this;
+  window.NavBar = window.NavBar || init();
+
+  return window.NavBar;
+
+  /**
+   * Init navbar
+   * @returns {NavBar}
+   */
+  function init() {
+    let navObj = {};
+
+    /*self.back = noop;
+    self.ignoreUrls = [];
+    self.hideOnUrls = [];*/
+    self.nav = $( '.jjb-navbar' );
+    self.backButton = $( '.jjb-navbar--back' );
+    self.backButton.click( onBack );
+    self.actionsButtons = $( '.jjb-navbar--action-buttons' );
```

Comment:
This seems to be an element, not a modifier. This could be jjb-navbar__action-buttons